css/_uswds-theme-custom-styles.scss

```diff
@@ -1430,7 +1430,17 @@ code[class*="language-plaintext"] {
 
 // External link styles
 #{$external-href} {
-  @include external-link(external-link, external-link-hover);
+  @include external-link(
+    usa-icons-bg/launch--blue-60v,
+    usa-icons-bg/launch--blue-70v
+  );
+}
+
+.usa-link--alt#{$external-href} {
+  @include external-link(
+    usa-icons-bg/launch--gray-5,
+    usa-icons-bg/launch--white
+  );
```

Comment:
This adds icons of the proper color to non-.gov links. (There are existing rules that hide these icons on the nav, buttons, etc)